xml/admin_monitoring_alerting.xml

```diff
@@ -1,1058 +1,374 @@
-    </para>
-    <itemizedlist>
-     <listitem>
+    <note>
       <para>
-       DingTalk (https://github.com/timonwong/prometheus-webhook-dingtalk)
+        The &prometheusmgr; needs to be restarted for any of the following
+        configuration changes to be applied.
       </para>
-     </listitem>
-     <listitem>
+    </note>
+    <sect2 xml:id="monitoring-http-requests">
+      <title>Configuring The Network Interface</title>
       <para>
-       IRC Bot (https://github.com/multimfi/bot)
+        By default, the &prometheusmgr; accepts HTTP requests on port 9283 on
+        all IPv4 and IPv6 addresses on the host. The port and listen address are
+        both configurable with <option>ceph config-key set</option> , with keys
+        <option>mgr/prometheus/server_addr</option> and
+        <option>mgr/prometheus/server_port</option> . This port is registered
+        with&prometheus;'s registry.
       </para>
-     </listitem>
-     <listitem>
       <para>
-       JIRAlert (https://github.com/free/jiralert)
+        To update the <literal>server_addr</literal> execute the following
+        command:
       </para>
-     </listitem>
-     <listitem>
+      <screen>
+&prompt.cephuser;ceph config set mgr mgr/prometheus/server_addr <replaceable>0.0.0.0</replaceable>
+      </screen>
       <para>
-       Phabricator / Maniphest (https://github.com/knyar/phalerts)
+        To update the <literal>server_port</literal> execute the following
+        command:
       </para>
-     </listitem>
-     <listitem>
+      <screen>
+&prompt.cephuser;ceph config set mgr mgr/prometheus/server_port <replaceable>9283</replaceable>
+      </screen>
+    </sect2>
+    <sect2 xml:id="monitoring-scrape-intervals">
+      <title>
+        Configuring <literal>scrape_interval</literal>
+      </title>
       <para>
-       prom2teams: forwards notifications to Microsoft Teams
-       (https://github.com/idealista/prom2teams)
+        By default, the &prometheusmgr; is configured with a scrape interval of
+        15 seconds. We do not recommend to use a scrape interval below 10
+        seconds. To set a different scrape interval in the &prometheus; module,
+        set <literal>scrape_interval</literal> to the desired value:
       </para>
-     </listitem>
-     <listitem>
+      <important>
+        <para>
+          The <literal>scrape_interval</literal> of this module should always be
+          set to match the &prometheus; scrape interval to work properly and not
+          cause any issues.
+        </para>
+      </important>
+      <screen>
+&prompt.cephuser;ceph config set mgr mgr/prometheus/scrape_interval <replaceable>15</replaceable>
+      </screen>
+    </sect2>
+    <sect2 xml:id="monitoring-stale-cache">
+      <title>Configuring The Cache</title>
       <para>
-       SMS: supports multiple providers
-       (https://github.com/messagebird/sachet)
+        On large clusters (&lt;1000 OSDs), the time to fetch the metrics may
```

Comment:
How about "more than" instead of the symbol? 

And should it not be a greater-than symbol, not a less-than symbol, anyway? I don't know what constitutes a large cluster, but surely this is a _lower_ bound not an upper bound?

Reply:
Oh that might be my bad, wrong sign